dask_match/expr.py

```diff
@@ -112,38 +112,42 @@ def __str__(self):
     def __repr__(self):
         return str(self)
 
-    def _tree_repr_lines(self, indent=0):
-        header = funcname(type(self)) + ":"
+    def _tree_repr_operand(self, i, lines, header, collapse_fusion=True):
```

Comment:
Can you say a bit more about the need to make the second method here?  

I briefly tried to implement Fused earlier, but gave up.  My approach at the time was to implement `_tree_repr_lines` on that class differently.  My preference would be if we could handle this with only one method, rather than have to split logic between two (this seems more complex to me).  Is that possible?